jsapp/js/components/RESTServices/RESTServicesForm.tsx

```diff
@@ -440,7 +487,7 @@ export default class RESTServicesForm extends React.Component {
                 type='text'
                 placeholder={t('https://')}
                 value={this.state.endpoint}
-                errors={this.state.endpointError}
+                errors={this.state.endpointError || undefined}
```

Comment:
this looks weird, let's just set `endpointError` to `undefined` instead of `null` in the first place?